processes/tests/.words

```diff
@@ -1,5 +1,5 @@
-OpenEO-based
-CropSAR
-OpenEO
-Sentinel-1
-chunk_polygon
\ No newline at end of file
+*OpenEO-based
```

Comment:
It should work without any special characters at the beginning. If it doesn't pick them up there might be another issue in the tooling.

Reply:
I tried them without any special characters or with . and this was with *

Reply:
It was a casing issue... openEO != OpenEO ;-)